doc/en/components/general-changelog-dv-blazor.md

```diff
@@ -10,6 +10,81 @@ namespace: Infragistics.Controls.Charts
 
 All notable changes for each version of {ProductName} are documented on this page.
 
+## **{PackageVerChanges-24-1-AUG}**
+
+### General
```

Comment:
would be nice to add links to the new components respective topics here.

Reply:
@skrustev